decidim-forms/app/views/decidim/forms/questionnaires/show.html.erb

```diff
@@ -3,154 +3,40 @@
-                          </div>
-                        <% end %>
+<%= render layout: "layouts/decidim/shared/layout_center" do %>
 
-                        <div class="row column tos-agreement">
-                          <%= form.check_box :tos_agreement, label: t(".tos_agreement"), id: "questionnaire_tos_agreement", disabled: !current_participatory_space.can_participate?(current_user) %>
-                          <div class="help-text">
-                            <%= decidim_sanitize_editor translated_attribute questionnaire.tos %>
-                          </div>
-                        </div>
-                      <% end %>
+  <%= render partial: "decidim/shared/component_announcement" if current_component.manifest_name == "surveys" %>
 
-                      <%= cell(
-                        "decidim/forms/step_navigation",
-                        step_index,
-                        total_steps: @form.total_steps,
-                        button_disabled: !current_participatory_space.can_participate?(current_user),
-                        form: form
-                      ) %>
-                    </div>
-                  <% end %>
-                <% end %>
-              </div>
-            <% end %>
-          <% else %>
-            <div class="answer-questionnaire">
-              <h3 class="section-heading"><%= t(".answer_questionnaire.title") %></h3>
-              <p>
-                <%= t(".answer_questionnaire.anonymous_user_message", sign_in_link: decidim.new_user_session_path, sign_up_link: decidim.new_user_registration_path).html_safe %>
-              </p>
+  <h1 class="h1 decorator my-12"><%= translated_attribute questionnaire.title %></h1>
 
-              <ol>
-                <%= cell("decidim/forms/question_readonly", collection: @questionnaire.questions.not_conditioned) %>
-              </ol>
-            </div>
-          <% end %>
-        <% else %>
-          <div class="section">
-            <div class="callout warning">
-              <h3 class="heading4"><%= t(".questionnaire_closed.title") %></h3>
-              <p><%= t(".questionnaire_closed.body") %></p>
-            </div>
-          </div>
-        <% end %>
-      </div>
-    </div>
+  <div class="text-xl text-gray-2 mb-10">
+    <%= decidim_sanitize_editor translated_attribute questionnaire.description %>
   </div>
-</div>
 
-<% content_for :js_content do %>
-  <%= javascript_pack_tag "decidim_forms" %>
+  <% unless questionnaire_for.try(:component)&.try(:published?) %>
+    <%= cell("decidim/announcement", t("decidim.forms.questionnaires.show.questionnaire_not_published.body"), callout_class: "warning") %>
+  <% end %>
+
+  <% if allow_answers? %>
+    <% if visitor_can_answer? %>
+      <% if visitor_already_answered? %>
+        <%= cell("decidim/announcement", { title: t("decidim.forms.questionnaires.show.questionnaire_answered.title"), body: t("decidim.forms.questionnaires.show.questionnaire_answered.body") }) %>
```

Comment:
This message is shown without any margin at the bottom of the view making it look weird.

![Margin under announcement](https://user-images.githubusercontent.com/864340/186687892-e281e4ca-c16d-4ac9-81d9-f06e0dc04558.png)

Note that @Crashillo did some changes regarding the page layouts and margins at #9755 and #9457. Maybe this one was affected too by those changes?